### PR TITLE
Patch svf example

### PR DESCRIPTION
**Pull request by beserge**

### CODE REVIEW

File: patch/Svf/Svf.cpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    
+    //setup controls
+    cutoff_ctrl.Init(patch.controls[0], 20, 20000, Parameter::LOGARITHMIC);
+    res_ctrl.Init(patch.controls[1], .3, 1, Parameter::LINEAR);
+    drive_ctrl.Init(patch.controls[2], .3, 1, Parameter::LINEAR);
+
+    //Put controls onscreen
+    patch.display.WriteString("Cut  Res  Drive", Font_7x10, true);
+    patch.display.SetCursor(0,50);
+    patch.display.WriteString("HP  LP  BP  Notch", Font_7x10, true);
```
Comment (tnorris) [inline patch/Svf/Svf.cpp:53]: I was playing with your filter and found a small bug, here's a fix :)
```suggestion
     patch.display.WriteString("LP  HP  BP  Notch", Font_7x10, true);
```

Comment (beserge) [reply patch/Svf/Svf.cpp:53]: Fixed, thanks!

**Comment by stephenhensley** [comment]
Solid.

**Comment by stephenhensley** [comment]
Though, we should do the same fix for the String/char* issue. I don't like the warnings.

I'll open an issue specifically for that.

**Pull request merged**